### DISCUSSION TITLE
For branch deployment, do we need to add details for the cloned database or does it automatically infer it from dbt?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17516] [Q&A] [answered]
Hey got a couple of questions, for the branch deployment do we need to add details for the cloned database or does it automatically infer it from dbt? We use dbt to put everything in snowflake

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C02LJ7G0LAZ/p1698761522664869?thread_ts=1698761522.664869&cid=C02LJ7G0LAZ).*

**Comment by shalabhc** [comment, marked as answer]
This depends a lot on how you are creating the cloned db. For the branch deployment typically you will have to create the cloned database yourself and point your code to use the clone when running in a branch. There is an example here on how to auto create the clone for branch deployments: https://docs.dagster.io/guides/dagster/branch_deployments#step-4-create-our-database-clone-upon-opening-a-branch
